src/Solr/Query/SearchQueryBuilder.php

```diff
@@ -114,6 +114,17 @@ protected function getResultsConfig()
         return $this->resultsConfig;
     }
 
+    public function build()
+    {
+        return new Query(
+            $this->getStoreId(),
+            $this->getQueryText(),
+            0,
+            $this->getPagination()->getPageSize() * $this->getPagination()->getCurrentPage(),
+            $this->getParamsBuilder()->setBroaden($this->broaden)->buildAsArray($this->getAttributetoReset())
```

Comment:
Is there a conceptual difference between `$broaden` and `$attributesToReset`, that justifies passing one via setter and one via parameter to the build method? It looks confusing.

If no, I think, the setter is the better way, as it fits the builder pattern better:

1. configure the builder
2. build the object based on builder configuration

Then  it could look like this:

```diff
            $this->getParamsBuilder()
                ->setBroaden($this->broaden)
-               ->buildAsArray($this->getAttributetoReset())
+               ->setAttributesToReset($this->getAttributetoReset())
+               ->buildAsArray()
```